hbase-http/src/main/java/org/apache/hadoop/hbase/http/HttpServer.java

```diff
@@ -652,6 +654,16 @@ private void initializeWebServer(String name, String hostName, Configuration con
         addFilterPathMapping(path, webAppContext);
       }
     }
+    // Check if disable stack trace property is configured
+    if (null != conf.get(HTTP_UI_SHOW_STACKTRACE_KEY, null)) {
```

Comment:
Should just do line 660 before this `if` check, then this would simply be ` if (showStackTraces) {`

Reply:
what I mean is you don't need to do `conf.get(HTTP_UI_SHOW_STACKTRACE_KEY, null)`. Just do `boolean showStackTraces = conf.getBoolean(HTTP_UI_SHOW_STACKTRACE_KEY, false);` before this if check. If HTTP_UI_SHOW_STACKTRACE_KEY is not set, your `showStackTraces` variable would be set to false and you can use it in the `if` check.

Reply:
did the change, by default showStackTraces would be true and only when it becomes false, we will need that logic to disable stack traces